locale/Makefile

```diff
@@ -42,18 +43,20 @@ uniq-po:
 	done
 
 tx-pull: $(EDITFILES)
-	tx pull -f
+	cd .. && tx pull -f
 	for f in $(EDITFILES) ; do \
 		sed -i 's/^\("Project-Id-Version: \).*$$/\1$(DOMAIN) $(VERSION)\\n"/' $$f; \
 	done
 
 tx-update: tx-pull
 	@echo
-	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation, then make -C locale mo-files to finish
+	@echo Run rake plugin:gettext[$(DOMAIN)] from the Foreman installation
+	@echo then run rake plugin:po_to_json[$(DOMAIN)] from the Foreman installation
+	@echo then run make -C locale mo-files to finish
 	@echo
 
 mo-files: $(MOFILES)
-	git add $(POFILES) $(POTFILE) ../locale/*/LC_MESSAGES
+	git add $(POFILES) $(POTFILE) $(JSFILES) ../locale/*/LC_MESSAGES
```

Comment:
Right now `.mo` files are in `.gitignore`. I think we should commit them to the repo too.